proto.h

```diff
@@ -684,9 +684,11 @@ PERL_CALLCONV_NO_RET void	Perl_croak_caller(const char* pat, ...)
 			__attribute__format__null_ok__(__printf__,1,2);
 #define PERL_ARGS_ASSERT_CROAK_CALLER
 
+#ifndef PERL_NO_INLINE_FUNCTIONS
```

Comment:
I presume this change is due to a bug fix in the regen file

Reply:
yep, it's a (unintended) side-effect of replacing `eq 'PERL_STATIC_INLINE'` with `=~ /PERL_STATIC(?:_FORCE)?_INLINE/`